homeassistant/components/homekit/util.py

```diff
@@ -36,6 +36,7 @@ def validate_entity_config(values):
 def show_setup_message(bridge, hass):
     """Display persistent notification with setup information."""
     pin = bridge.pincode.decode()
+    _LOGGER.info('Pincode: %s', pin)
```

Comment:
Info is the default logging message. Do we really always want to log it?

Reply:
Since it is in 'show_setup_message', it is only being logged if HomeKit isn't paired. Isn't the default for most people 'warning'? If I set this to debug, it might become confusion since that would generate quite a lot of other messages.